docs/framework/serialization/how-to-serialize-an-object-as-a-soap-encoded-xml-stream.md

```diff
@@ -65,14 +65,14 @@ manager: "erikre"
   
 ```vb  
 ' Serializes a class named Group as a SOAP message.  
-Dim myTypeMapping As XmlTypeMapping = (New SoapReflectionImporter(). _  
+Dim myTypeMapping As XmlTypeMapping = New SoapReflectionImporter(). _  
 ImportTypeMapping(GetType(Group))  
```

Comment:
I think this line should be indented, to make it clearer that it's not a separate statement. Do you want to add that change to this PR?

Alternatively, you could just put the statement on a single line, like the C# version does.